.github/workflows/build.yml

```
@@ -13,19 +13,27 @@ jobs:
     tests:
         runs-on: ubuntu-latest
 
-        name: "PHP ${{ matrix.php }}, MySQL ${{ matrix.mysql }}, Node ${{ matrix.node }}"
+        name: "Sylius ${{ matrix.sylius }}, PHP ${{ matrix.php }}, Symfony ${{ matrix.symfony }}, MySQL ${{ matrix.mysql }}"
 
         strategy:
             fail-fast: false
             matrix:
                 php: [7.4, 7.3]
+                symfony: [^4.4, ^5.2]
+                sylius: [~1.8.0, ~1.9]
```

I would be for:
```suggestion
                sylius: [~1.8.0, ~1.9.0]
```
to add next minor in clear way to our tests